apps/src/templates/certificates/Congrats.jsx

```diff
@@ -57,6 +59,8 @@ export default function Congrats(props) {
         hideDancePartyFollowUp={hideDancePartyFollowUp}
       />
       {userType === 'signedOut' && isEnglish && <TeachersBeyondHoc />}
+      <hr style={styles.divider} />
+      <PetitionCallToAction gaPagePath={`/congrats/${tutorial}`} />
```

Comment:
channeling PM for a moment... I I think it will be important that we can distinguish between petition clicks on pegasus vs dashboard. I would suggest the following 2 changes, here and in https://github.com/code-dot-org/code-dot-org/blob/4261dc2260a799aa9cc6157281337808ae955874/apps/src/templates/certificates/petition/PetitionForm.jsx#L57-L59
1. change `petition` to `studio_petition` in PetitionForm
2. pass in `tutorial` here instead of `gaPagePath`. then in PetitionForm, specify `{tutorial: tutorial}` instead of `{page: gaPagePath}`

Reply:
Hmm, yes––good call. Will update!